tile/targets/cpu/executable.cc

```diff
@@ -79,23 +80,31 @@ void Executable::Run(const std::map<std::string, void*>& buffers) {
   }
   void* argvec = args.data();
   uint64_t entrypoint = engine_->getFunctionAddress(invoker_name_);
+  // To get the raw execution time for generated code.
+  // struct timeval tp;
+  // gettimeofday(&tp, NULL);
+  // long long int ms = tp.tv_sec * 1000000 + tp.tv_usec; //get current timestamp in microseconds
   ((void (*)(void*))entrypoint)(argvec);
+  // gettimeofday(&tp, NULL);
+  // long long int msDone = tp.tv_sec * 1000000 + tp.tv_usec; //get current timestamp in microseconds
+  // IVLOG(1, "Raw execution time: " << msDone - ms);
```

Comment:
Is this commented code still necessary? If it's still useful, let's find a way to toggle it when appropriate.

Reply:
Is there any way to retain this code without it being a comment? Perhaps something like an env var to enable it? Or is it obsolete now?

Reply:
Discussed with Jeremy... We dcided to remove the code for now.